src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -240,30 +249,35 @@ class TensorRTRuntime : public JSONRuntimeBase {
     helper.DeclareField("inputs", &engine_and_context.inputs);
     helper.DeclareField("outputs", &engine_and_context.outputs);
     helper.ReadAllFields(&reader);
-    trt_engine_cache_[symbol_name_] = engine_and_context;
+    const int batch_size = 1;
```

Comment:
We should probably add the batch size of the engine in the key (file name) of the cached engine.